### ISSUE TITLE
Merge packages/components to packages/cli?

### ISSUE THREAD

**Issue by severo** [hyparam/hyperparam-cli#119]
It's complex to try the current code of components within the cli. One way to do this is:

- in components, run `npm pack` -> it generates hyparam-components-0.1.9.tgz
- in cli/package.json, replace `"@hyparam/components": "0.1.9",` with `"@hyparam/components": "../components/hyparam-components-0.1.9.tgz",`
- then `npm update @hyparam/components`

We could move components back to cli, and publish everything under the `hyperparam` package (ie deprecate @hyparam/components). The package would be bigger, but thanks to tree shaking, it should not affect too much dependent packages if they use a build step.

If we do so: should we move the apps/ subdirectory elsewhere?

**Comment by severo** [comment]
Let's put on hold while trying if npm workspaces help (#130)

**Comment by platypii** [comment]
I think we should just merge components into cli, and move the unified package up to the top level again. I'm not convinced that the separation of components into workspaces is worth the hassle. I still get weird issues with npm install, and the eslint config is hard to follow. Plus my workflow for editing components now has two tabs and a random series of `npm run build` commands. One package would make a much better experience.

We should export the individual components in the `hyperparam` package and update `demos` and `space` to depend on `hyperparam` instead of `@hyparam/components`. Then we should deprecate components package.

**Comment by severo** [comment]
Yes, I agree. After all, it was not worth it, in particular now that the demos live in another repo.
